pkgs/tools/system/uefitool/new-engine.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  qtbase,
+  cmake,
+  wrapQtAppsHook,
+  zip,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "uefitool";
+  version = "A71";
+
+  src = fetchFromGitHub {
+    sha256 = "sha256-NRlrKm5+eED6oyvFRSEhn0EUbMsPJtuFAyv3vgY/IUI=";
+    owner = "LongSoft";
+    repo = "uefitool";
+    tag = version;
+  };
+
+  buildInputs = [ qtbase ];
+  nativeBuildInputs = [
+    cmake
+    zip
+    wrapQtAppsHook
+  ];
+  patches = lib.optionals stdenv.isDarwin [ ./bundle-destination.patch ];
+
+  meta = {
+    description = "UEFI firmware image viewer and editor";
+    homepage = "https://github.com/LongSoft/uefitool";
+    license = lib.licenses.bsd2;
+    maintainers = with lib.maintainers; [ athre0z ];
+    platforms = lib.platforms.linux ++ lib.platforms.darwin;
```

```suggestion
    platforms = lib.platforms.unix;
```
Might `unix` make more sense here? in practice that means darwin+linux as that is what our CI covers. However, it doesn't straight disregard the possibility of running this on e.g. bsd.